components/homme/src/share/prim_driver_base.F90

```diff
@@ -1435,7 +1457,7 @@ subroutine applyCAMforcing_tracers(elem,hvcoord,np1,np1_qdp,dt,adjustment)
                   elem%state%Qdp(i,j,k,q,np1_qdp) = elem%state%Qdp(i,j,k,q,np1_qdp)+v1
                   if (q==1) then
                      elem%derived%FQps(i,j)=elem%derived%FQps(i,j)+v1/dt
-                     dp_adj(i,j,k)=dp_adj(i,j,k) + v1/dt
+                     dp_adj(i,j,k)=dp_adj(i,j,k) + v1
```

Comment:
is it true that if !use_moisture and !adjust_ps, then dp3d is still adjusted for vapor?

Reply:
thanks - fixed.